### PR TITLE
fix(LI): adjust insightCard height in demo

### PR DESCRIPTION
**Pull request by Esorakouki**
### PR includes

- [ ] fixed #0
- [ ] add / modify test cases
- [ ] documents, demos

### CODE REVIEW

**Review by neoddish — APPROVED** [review]
LGTM

File: sites/ava-site/examples/lite-insight/custom/demo/impact.jsx [hunk 1]
```diff
@@ -47,7 +47,8 @@ const App = () => {
       <h2 style={{ borderBottom: '1px solid #e9e9e9' }}>Insight list</h2>
       <Spin spinning={loading} style={{ marginTop: 80 }}>
         <div style={{ width: '100%' }}>
-          {result.insights && result.insights.map((item, index) => <InsightCard key={index} insightInfo={item} />)}
+          {result.insights &&
```
Comment (pddpd) [inline sites/ava-site/examples/lite-insight/custom/demo/impact.jsx:50]: How about changing it to `result.insights?.map((item, index) => <InsightCard key={index} insightInfo={item} height={400} />)`. There are two more below.

File: sites/ava-site/examples/lite-insight/custom/demo/homogeneous.jsx [hunk 2]
```diff
@@ -41,7 +41,9 @@ const App = () => {
       <Spin spinning={loading} style={{ marginTop: 80 }}>
         <div style={{ width: '100%' }}>
           {result.homogeneousInsights &&
```
Comment (pddpd) [inline sites/ava-site/examples/lite-insight/custom/demo/homogeneous.jsx:43]: How about changing it to `result.homogeneousInsights?.map((item, index) => (<InsightCard key={index} insightInfo={item} height={400} />))`

**Pull request merged**